ChangeSet
  1.1794 04/04/14 11:09:46 anton@samba.org[torvalds] +1 -0
  [PATCH] large cpumask fix
  
  The RCU code was missing cpus_empty() in one place, required with large
  cpumasks.

  kernel/rcupdate.c
    1.10 04/04/14 08:28:51 anton@samba.org[torvalds] +1 -1
    large cpumask fix

diff -Nru a/kernel/rcupdate.c b/kernel/rcupdate.c
--- a/kernel/rcupdate.c	Wed Apr 14 21:36:05 2004
+++ b/kernel/rcupdate.c	Wed Apr 14 21:36:05 2004
@@ -182,7 +182,7 @@
 	 * it here
 	 */
 	spin_lock_irq(&rcu_ctrlblk.mutex);
-	if (!rcu_ctrlblk.rcu_cpu_mask)
+	if (cpus_empty(rcu_ctrlblk.rcu_cpu_mask))
 		goto unlock;
 
 	cpu_clear(cpu, rcu_ctrlblk.rcu_cpu_mask);
.........................................................................
# vim: syntax=diff

